From 5d46c26b4755df25daae69412bcc4133e8c71913 Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Wed, 22 Nov 2017 11:22:15 +0800
Subject: [PATCH 2882/5242] MLK-16919-6 dts: add property
 'fsl,two-8bits-sensor-mode' for CSI

commit  1b380d06ae01894d704d35daf3896a5ee83a6011 from
https://source.codeaurora.org/external/imx/linux-imx.git

add property 'fsl,two-8bits-sensor-mode' for CSI on iMX8MQ

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Reviewed-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 3208cfb..97480a9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -758,6 +758,7 @@
 
 &csi1_bridge {
 	fsl,mipi-mode;
+	fsl,two-8bit-sensor-mode;
 	status = "okay";
 
 	port {
@@ -769,6 +770,7 @@
 
 &csi2_bridge {
 	fsl,mipi-mode;
+	fsl,two-8bit-sensor-mode;
 	status = "disabled";
 
 	port {
-- 
1.7.9.5

